src/components/LocationTime/LocationTime.js

```diff
@@ -1,12 +1,33 @@
+import { DateTime } from "luxon";
+import { useState } from "react";
 import "./LocationTime.css";
 
 export function LocationTime({
-    time,
     location,
-    day,
-    date,
-    alignment
+    timezone = "local",
+    alignment = "left",
 }) {
+    let time = null;
+    let day = null;
+    let date = null;
+    
+    // By default, as local timezone is selected, we create date in local time
+    // let dt = DateTime.local();
+    let [ dt, setDateTime ] = useState(DateTime.local());
```

Comment:
make it const 
rename it as dt and setDt or dateTime and setDateTime

Reply:
Done